### ISSUE TITLE
Gateway response parsing is platform-dependent

### ISSUE THREAD

**Issue by alexminza** [maibank/maibapi#3]
See original issue: https://github.com/alexminza/wc-moldovaagroindbank/issues/29

They are running on a Windows server - thus `PHP_EOL` is defined differently and `explode(PHP_EOL...` does not properly parse the response from a *nix server.

https://github.com/maibank/maibapi/blob/main/src/MaibApi/MaibClient.php#LL77
```
$array1 = explode(PHP_EOL, trim((string)$response->offsetGet('additionalProperties')));
```

**Comment by alexminza** [comment]
@ChR1sMC suggests a great article here :) https://eidson.info/post/php_eol_is_broken

**Comment by ChR1sMC** [comment]
> @ChR1sMC suggests a great article here :) https://eidson.info/post/php_eol_is_broken

Thx @alexminza , I replaced the code of the library on my server, as article suggests:
To Replace:
`$lines = explode(PHP_EOL, $str);`

With:
`$lines = preg_split('/\n|\r\n?/', $str);`

And I can confirm that my issue has been fixed !

**Comment by alexminza** [comment]
> I replaced the code of the library on my server, as article suggests

@ChR1sMC can you please submit a PR for this?

**Comment by ChR1sMC** [comment]
Will do, thank you

**Comment by alexminza** [comment]
It seems that there was just a release with this modification implemented directly in the code, without any PR or issue mentioned.

https://github.com/maibank/maibapi/releases/tag/2.0.4

https://github.com/maibank/maibapi/compare/2.0.3...2.0.4

**Comment by alexminza** [comment]
@maib-ecomm please use branches and PR merging along with issues mentioning to allow proper tracking for the fixes.

**Comment by maib-ecomm** [comment]
Hi! Thank you!
Fixed in 2.0.4 / 3.0.4
